src/index.ts

```diff
@@ -1,3 +1 @@
-import builder from './builder'
-
-export = builder
+export {default} from './builder'
```

Comment:
Will this work? Will it result in `module.exports = builder`?

Reply:
Uh, good catch! That wasn't supposed to be included in this PR.

Reply:
Although, tsconfig.json says `module: es2015`, and my editor gives me an error saying:
```
TS1203: Export assignment cannot be used when targeting ECMAScript modules. Consider using 'export default' or another module format instead.
```

But yeah, that's for another PR.